vignettes/new-application.Rmd

```diff
@@ -59,7 +59,8 @@ add tabbed pages, etc.
 
 The right sidebar is, like the left sidebar, reserved for configuration options, settings, 
 and general user application controls and functionality that will affect the body of the
-application.  
+application.
+
```

Comment:
Added an empty line since the following text was already in a new line and new lines seem to be preceded with empty lines in this document. Please let me know if this empty line should be removed.